docker/configs/config-blob-dc1.yaml

```diff
@@ -28,64 +35,19 @@ dataCenter:
   # Should be identical to dataCenterServiceUri if this instance resides in the system data center
   systemDataCenterServiceUri: http://emodb-web-dc1:8080
 
-kafka:
-  bootstrapServers: kafka:9092
-  producer:
-    lingerMs: 5
-
-megabus:
-  applicationId: megabus-local
-  refResolverConsumerGroupName: emodb.megabus.resolver
-  delayProcessorConsumerGroupName: emodb.megabus.delay
-  megabusRefTopic:
-    name: emodb.internal.megabus.ref
-    partitions: 8
-    replicationFactor: 1
-  megabusTopic:
-    name: emodb.megabus
-    partitions: 8
-    replicationFactor: 1
-  missingRefTopic:
-    name: emodb.internal.megabus.missing
-    partitions: 8
-    replicationFactor: 1
-  retryRefTopic:
-    name: emodb.internal.megabus.retry
-    partitions: 8
-    replicationFactor: 1
-  boot:
-    queueServiceApiKey: local_admin
-    scanStatusTable: __system_megabus_boot
-    pendingScanRangeQueueName: megabus-boot-pending-scan-ranges
-    completeScanRangeQueueName: megabus-boot-complete-scan-ranges
-  refProducer:
-    batchSize: 400
-    skipWaitThreshold: 250
-    pollIntervalMs: 100
-
-
 
 systemOfRecord:
-  deltaBlockSizeInKb: 16
-  cellTombstoneCompactionEnabled: true
-  cellTombstoneBlockLimit: 2
-  stashRoot: s3://emodb-us-east-1/stash/ci
-
-  # Optional property - to exclude the tables satisfying the condition from the daily Stash run.
-  stashBlackListTableCondition: "intrinsic(\"~table\":like(\"nostash:*\"))"
 
   # How long should we retain historical deltas? To disable, use PT0S
   historyTtl: PT48H
-
+ 
```

Comment:
Could you please remove extra whitespace

Reply:
There is no extra whitespace.